.kuzzlerc.sample

```diff
@@ -778,6 +778,11 @@
           }
         }
       },
+      // Global settings to apply by default (can be overriden by user requests)
+      "globalSettings": {
+        "number_of_shards": 1,
+        "number_of_replicas": 1
```

Comment:
properties with underscore but all other properties in this file are kebab case

Reply:
This was THE potential debate of this PR 😄 
@kuzzleio/core what do you prefer? Vote on this comment with 🐍 for snake case (the current format) or 🐫 for camel case

Reply:
Since it's a native option for ES I prefer to keep the same syntax

Reply:
:snake: because even if it's not the style we use in kuzzle (coding standard)

It is still the syntax awaited by ES and i believe it should be enough to accept this format